source/sgp_mode/SGPDataNodes.h

```diff
@@ -246,14 +271,27 @@ void SGPWorld::SetupTasksNodes() {
   }
 }
 
-
+/**
+ * Input: None
+ *
+ * Output: Data monitor for the number of steal instruction ran
+ *
+ * Purpose: To check and edit the data for steal instructions
```

Comment:
This purpose is just to get the data node and create it if it doesn't already exist